app/stories/pages/eligibility.stories.tsx

```diff
@@ -1,14 +1,44 @@
+import { useState } from '@storybook/client-api'
 import { ComponentMeta, ComponentStory } from '@storybook/react'
 
 import EligibilityPage from '@pages/eligibility'
 
+import { getBackRoute, getForwardRoute } from '@utils/routing'
+
+import {
+  getEmptyMockSession,
+  getMockSessionData,
+} from '../../tests/helpers/mockData'
+
 export default {
   title: 'Pages/Eligibility',
   component: EligibilityPage,
+  argTypes: {
+    session: {
+      table: {
+        disable: true,
+      },
+    },
+  },
 } as ComponentMeta<typeof EligibilityPage>
 
-const Template: ComponentStory<typeof EligibilityPage> = () => (
-  <EligibilityPage previousRoute="review" />
-)
+const Template: ComponentStory<typeof EligibilityPage> = (args) => {
+  const [session, setSession] = useState(args.session)
+  args.session = session
+  args.setSession = setSession
```

Comment:
Note for reviewers: I'm using storybook's `useState()` in this story to hook up the controls because the page has the `session` and `setSession` variables passed in as props. Is it a bad practice to put a state into a storybook like this?